tools/ssh_helpers/hab-env

```diff
@@ -9,7 +9,7 @@ SESSION="hab_${HAB_ENV}"
 
 # (Re-)generate a tmuxinator session description. This ensures we're
 # always up-to-date
-~/bin/generate-habitat-env-tmuxinator.sh ${HAB_ENV} \
```

Comment:
Wonder if we should just leave the path prefix off altogether and tell people to put these utilities somewhere on their path instead.

Reply:
I found it intuitive (and requiring no setup) just to execute them in place -- although having them on the path would be convenient in a pinch.

Reply:
However people feel it's easier to work with; I leave it to your discretion 😄